pkg/kubelet/pod_workers.go

```diff
@@ -96,8 +97,11 @@ const (
 	// jitter factor for resyncInterval
 	workerResyncIntervalJitterFactor = 0.5
 
-	// jitter factor for backOffPeriod
+	// jitter factor for backOffPeriod and backOffOnNetworkNotReadyPeriod
 	workerBackOffPeriodJitterFactor = 0.5
+
+	// backoff period when network is not ready.
+	backOffOnNetworkNotReadyPeriod = time.Second
```

Comment:
We should generalize this for other non-pod-specific issues, but I think it's okay to do it later when there are more use cases.

Reply:
Done